javascript/javascript1/week3/array_removal.js

```diff
@@ -0,0 +1,18 @@
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmad";
+  
+  // Write some code here
+  const remove = names.splice(1, 1)
```

Comment:
although this is correct for this case.  can you think about the general case to solve the problem...by using `for loops` ?

Reply:
Yes I changed with for loop